core/shared/src/test/scala/fs2/StreamSuite.scala

```diff
@@ -1042,4 +1042,13 @@ class StreamSuite extends Fs2Suite {
     }
   }
 
+  test("rechunkRandomlyWithSeed should correclty rechunk big chunks at the end of a stream") {
```

Comment:
Looks like we keep those tests here.

https://github.com/typelevel/fs2/blob/5dbe235e5a4e763251f94c3880a75ba20915edbb/core/shared/src/test/scala/fs2/StreamCombinatorsSuite.scala#L1317